src/platform/site-wide/sass/shame.scss

```diff
@@ -589,3 +589,10 @@ img[data-srcset] {
     width: 100%;
   }
 }
+
+.all-link-arrow {
```

Comment:
Due to the new DOM hierarchy, the arrows are no longer receiving the styles in `all-link-arrow` from here https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/8d39616e957cb44b6664a658b2c51dca8076fe7e/packages/formation/sass/modules/_m-hub-page-link-list.scss

Reply:
I'd rather not touch the css in that file directly, not being confident that I won't be messing up styles in other places.

Reply:
The problem with this is that it will also affect the other all-link-arrows 🤔 could we make this a different class name?